WolvenKit.App/ViewModels/Shell/AppViewModel.WScript.cs

```diff
@@ -76,6 +76,19 @@ private async Task RunFileValidationOnProject()
             return;
         }
 
+        var txtFilesInResources = ActiveProject.ResourceFiles.Where(f => f.EndsWith(".txt"))
+            .Select(f => ActiveProject.GetRelativeResourcePath(f).GetResolvedText())
+            .Where(f => f is not null)
+            .Where(f => Path.GetExtension(f?.Replace(".txt", "")) is not null)
```

Comment:
It should check for string is not empty here not null.

Reply:
fixed